### ISSUE TITLE
About Pretraining Model

### ISSUE THREAD

**Issue by SunnyXudongShi** [gene-chou/EFPN-detectron2#2]
Hello! First of all, I think your idea is very good. I read your blog, and I want to Recurrence results myself, but I didn't find the pretraining model. Can you provide the pre training model on Baidu cloud.

**Comment by gene-chou** [comment]
We have not implemented the loss function described in the original author's paper so the EFPN is not trained properly. We only provide the EFPN structure in this repo.
